setup.py

```diff
@@ -29,10 +29,12 @@
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
     ],
     install_requires=[
         'panoramic-auth>=0.2.0',
         'requests>=2.24.0',
+        'urllib3>=1.26.1',
```

Comment:
I had to add this in because the newest release of requests (2.25.0) caused an error and the Actions runners wouldn't pick up the upgrade. I assume we can remove this at some point in the future (or we could just lock our version of requests at 2.24.0), but this worked for now...